doc/sphinx/language/gallina-extensions.rst

```diff
@@ -85,8 +83,8 @@ To build an object of type :n:`@ident`, one should provide the constructor
 .. _record-named-fields-grammar:
 
   .. productionlist::
-    term : {| [`field_def` ; … ; `field_def`] |}
-    field_def : name [binders] := `term`
+    record_term : {| [`field_def` ; … ; `field_def`] |}
+    field_def : name [binders] := `record_term`
```

Comment:
I think `term` was really meant there and the above should have been an *extension* of the term syntax.
If that helps, I think that this should not be presented in the extension chapter but in the specification chapter anyways, as part of the main grammar.